test/integration/hostgroup_js_test.rb

```diff
@@ -27,6 +27,7 @@ class HostgroupJSTest < IntegrationTestWithJavascript
     host = Hostgroup.where(:name => "myhostgroup1").first
     assert host
     assert_equal env.name, host.environment.name
+    assert page.has_current_path? hostgroups_path
```

Comment:
shouldn't it be `have_current_path`?
I think `has_current_path?` returns `true` or `false` while `have_current_path` returns the current URL (after waiting).